ldaptor/protocols/pureldap.py

```diff
@@ -1434,7 +1451,12 @@ def __init__(self, requestName=None,
 
     def __repr__(self):
         l = []
-        # TODO userIdentity, oldPasswd, newPasswd
+        if self.userIdentity is not None:
+            l.append('userIdentity={}'.format(repr(self.userIdentity)))
+        if self.oldPasswd is not None:
+            l.append('oldPasswd={}'.format(repr(self.oldPasswd)))
+        if self.newPasswd is not None:
+            l.append('newPasswd={}'.format(repr(self.newPasswd)))
```

Comment:
I don't know why this is TODO... but I think than unless there is an explicit request, is better to leave the password outside of the repr... is ok to have userIdentity... but maybe use a private member `self._userIdentity`